gulpfile.js

```diff
@@ -146,6 +146,9 @@ gulp.task('babelwatch', function() {
     //todo: test if this even works... also, maybe we can condense these two watches into one line:
     //gulp.watch('src/client/**/*.js*', ['babel']);     ?
     gulp.watch('src/client/**/*.js', ['babel']);
+
+    // Watch server files as well
+    gulp.watch('src/server/*.js', ['babel']);
```

Comment:
I think this should be `src/server/**/*.js` so that it gets all the js files in all subdirectories as well.